llm_bench/python/benchmark.py

```diff
@@ -187,21 +191,30 @@ def run_text_generation_benchmark(model_path, framework, device, args, num_iters
     return iter_data_list, pretrain_time
 
 
-def run_image_generation(input_text, nsteps, num, image_id, pipe, args, iter_data_list):
+def run_image_generation(image_param, num, image_id, pipe, args, iter_data_list):
     set_seed(args['seed'])
-    log.info(f'batch_size={args["batch_size"]}')
+    input_text = image_param['prompt']
+    image_width = image_param.get('width', DEFAULT_IMAGE_WIDTH)
+    image_height = image_param.get('height', DEFAULT_IMAGE_HEIGHT)
+    nsteps = image_param.get('steps', DEFAULT_INFERENCE_STEPS if 'lcm' not in args["model_name"] else 4)
```

Comment:
If this 4 is a default value for something?

Reply:
Define LCM_DEFAULT_INFERENCE_STEPS for 4 which is infer steps of lcm models.